packages/components/src/Components/ErrorState/ErrorState.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Stack, StackItem } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons/';
+import DefaultErrorMessage from './DefaultErrorMessage';
+import propTypes from 'prop-types';
+import {
+    Title,
+    EmptyState,
+    EmptyStateVariant,
+    EmptyStateIcon,
+    EmptyStateBody,
+    Button
+} from '@patternfly/react-core';
+
+const ErrorState = ({ errorTitle, errorDescription, redirectLink }) => {
```

Comment:
Might be good to take rest of the `props`
```suggestion
const ErrorState = ({ errorTitle, errorDescription, redirectLink, ...props }) => {
```